File: Chapter-2/Java/Problem2.java

//import libraries
import java.util.Scanner;

public class Problem2 {
    public static void main(String[] args) {
        double x, y, r, phi;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter x:");
        x = sc.nextDouble();
        System.out.println("Enter y:");
        y = sc.nextDouble();
        r = Math.sqrt((Math.pow(x, 2) + Math.pow(y, 2)));
        phi = Math.atan(y / x);

        System.out.printf("Cartesian Coordinates: (%.3f,%.3f) \n", x, y);
        System.out.printf("Polar Coordinates: (%.3f,%.3f)", r, phi);
        sc.close();
    }
}

File: Chapter-2/Python/Problem2.py
import math as m
# take input
x,y = list(map(float,input().split()))
#calculate polar coordinates
r = m.sqrt(x**2+y**2)
phi = m.atan(y/x)
# display the results
print("Cartesian Coordinates: (%.3f,%.3f)"%(x,y))
print("Polar Coordinates: (%.3f,%.3f)" %(r,phi))